metaflow/datastore/content_addressed_store.py

```diff
@@ -124,29 +125,28 @@ def load_blobs(self, keys, force_raw=False):
         Dict: string -> bytes:
             Returns the blobs as bytes
         """
-        to_load = []
         results = {}
-        if self._blob_cache is not None:
-            for k in keys:
-                v = self._blob_cache.load(k)
-                if v is not None:
-                    results[k] = v
-                else:
-                    to_load.append(k)
-        else:
-            to_load = keys
-        to_load_paths = [
-            self._backend.path_join(self._prefix, k[:2], k) for k in to_load]
-        with self._backend.load_bytes(to_load_paths) as load_results:
-            for k, path in zip(to_load, to_load_paths):
+        load_paths = []
+        for key in keys:
+            blob = None
+            if self._blob_cache:
+                blob = self._blob_cache.load(key)
+            if blob:
```

Comment:
I would keep the `is not None` to allow for empty lists, etc.

Reply:
`_blob_cache` needs to be of type `content_addressed_store.BlobCache`, so it can be false only if not specified

Reply:
Sorry, I meant the `if blob` part.

Reply:
Reverted